frame/contracts/src/tests.rs

```diff
@@ -143,12 +150,12 @@ impl Default for TestExtension {
 }
 
 impl ChainExtension<Test> for TestExtension {
-	fn call<E>(func_id: u32, env: Environment<E, InitState>) -> ExtensionResult<RetVal>
+	fn call<E>(&mut self, func_id: u32, env: Environment<E, InitState>) -> ExtensionResult<RetVal>
```

Comment:
Might also be worth renaming `func_id` since it now has two roles

Reply:
Makes sense. I removed the parameter all together. See above.